tf/service_env_vars.tf

```diff
@@ -12,8 +12,10 @@ variable "service_env_vars" {
     # Ports
 
     ## BwdServer
-    "DARK_CONFIG_BWDSERVER_BACKEND_PORT"    = "11001"
-    "DARK_CONFIG_BWDSERVER_KUBERNETES_PORT" = "11002"
+    "DARK_CONFIG_BWDSERVER_BACKEND_PORT"           = "11001"
+    "DARK_CONFIG_BWDSERVER_KUBERNETES_PORT"        = "11002"
+    "DARK_CONFIG_BWDSERVER_DANGER_BACKEND_PORT"    = "11001"
+    "DARK_CONFIG_BWDSERVER_DANGER_KUBERNETES_PORT" = "11002"
```

Comment:
I guess we need to assign these to make Config.fs up the chain not break?

Reply:
Right. We can remove them later